docs/api-docs/management-apis/contextual-filters.md

```diff
@@ -0,0 +1,310 @@
+# Contextual Filters
+
+<div class="otp" id="no-index">
+
+### On this page
+- [View pre-configured filters](#view-pre-configured-filters)
+- [View available filters](#view-available-filters)
+- [Configure contextual filters](#configure-contextual-filters)
+- [Related resources](#related-resources)
+
+</div>
+
+The [Settings API](https://developer.bigcommerce.com/api-reference/store-management/settings) allows you to manage settings and configurations for BigCommerce hosted stores and headless storefronts. 
+
+You can use the [contextual filters](insert-link-here) endpoint of the Settings API to programmatically configure a store's Product Filtering feature.
```

Comment:
I have a habit of mentioning that our feature "Product Filtering" is often known as "[faceted search](https://en.wikipedia.org/wiki/Faceted_search)" in the broader e-commerce world. It might be good to do so here as well. 🍹